.github/workflows/ci.yml

```diff
@@ -18,15 +18,18 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: 2.7
+          python-version: 3.8
```

Comment:
Nice! I guess we have this in other examples?

Reply:
Yes, we have `python2.7` in several demos such as `world_population` and `root6-roofit`. We also need to change the validation in some of them take off the `| grep -q 'workflow validates' ` and to add all the ´.yaml´ files in other ones. It's a little sprint to update them all. I'll make the corresponding PRs set to update all the demos